edx_django_utils/hooks/exceptions.py

```diff
@@ -0,0 +1,16 @@
+"""
+Exceptions thrown by Hooks.
+"""
+
+
+class HookException(Exception):
+    """
+    Base exception for hooks. It is re-raised by the Pipeline Runner if any of
+    the actions/filters that is executing raises it.
+    """
+
+    def __init__(self, message='', redirect_to=None, error_code=None):
```

Comment:
error_code or status_code?

Reply:
I took error_code from https://github.com/edx/edx-platform/blob/3d19ee8272bd11eb5c80338672e6c504388de349/openedx/core/djangoapps/user_authn/exceptions.py#L7 but maybe we must follow https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html